src/test/groovy/org/osiam/storage/query/GroupFilterExpressionSpec.groovy

```diff
@@ -0,0 +1,60 @@
+package org.osiam.storage.query
+
+import org.osiam.resources.scim.Group
+import spock.lang.Specification
+import spock.lang.Unroll
+
+class GroupFilterExpressionSpec extends Specification {
+
+    @Unroll
+    def 'QueryField "#queryField" can be resolved'() {
+        given:
+        def expression = new GroupFilterExpression("${queryField}", FilterConstraint.EQUALS, 'irrelevant')
+
+        expect:
+        expression.field.queryField.toString() == queryField
+        expression.field.getUrn() == Group.SCHEMA
+        !expression.field.isExtension()
+
+        where:
+        queryField << GroupQueryField.values().collect { it.toString() }
+    }
+
+    @Unroll
+    def 'Resolving fully qualified query field "#queryField"'() {
+        given:
+        def expression = new GroupFilterExpression("${Group.SCHEMA}:${queryField}", FilterConstraint.EQUALS, 'irrelevant')
+
+        expect:
+        expression.field.queryField.toString() == queryField
+        expression.field.getUrn() == Group.SCHEMA
+        !expression.field.isExtension()
+
+        where:
+        queryField << GroupQueryField.values().collect { it.toString() }
+    }
+
+    @Unroll
+    def 'Resolving field "#queryField" separated by colon is possible'() {
```

Comment:
Wait a moment… Why should this be working? It didn't work before, and shouldn't work according to the SCIM spec.